### ISSUE TITLE
Make deleted messages disappear completely like in telegram

### ISSUE THREAD

**Issue by User1l0** [oxen-io/session-android#787]

**Comment by tashijayla** [comment]
Please give a option to delete a deleted message or reply, I don't want a deleted message to just disappear

**Comment by x80486** [comment]
Yeah, currently the message is just turned into a tag that says it was deleted; I think it's better to convey the actual action and just remove it completely without any traces.
